.envrc

```diff
@@ -125,12 +126,19 @@ export NODE_OPTIONS=--max-old-space-size=4096
 # Enable this by default for development envs (CI/deploys do not use envrc)
 export SENTRY_UI_HOT_RELOAD=1
 
+### You can override the exported variables with a .env file
+# All exports should happen before here unless they're safeguarded (see devenv error reporting below)
+if [ -f '.env' ]; then
```

Comment:
Moving this up here works better when I want to load "SENTRY_DEVENV_NO_REPORT" through the `.env` file.